web-ui/src/components/guild-results/EditGuildModal.jsx

```diff
@@ -211,7 +214,7 @@ const EditGuildModal = ({ guild = {}, open, onSave, onClose, headerText }) => {
           onChange={onGuildMembersChange}
           getOptionLabel={option => option.name}
           isOptionEqualToValue={(option, value) =>
-            value ? value.id === option.id : false
+            value && option.id === value.memberId
```

Comment:
I'm pretty sure we want to compare against the `memberId` property in the `value` object rather than the `id` property.